linux/ubuntu/22.04/foxpass_setup.py

```diff
@@ -146,7 +146,7 @@ def install_dependencies():
 
 
 def write_foxpass_ssh_keys_script(apis, api_key):
-    base_curl = 'curl -s -q -m 5 -f -H "Authorization: Token ${secret}" "%s/sshkeys/?user=${user}&hostname=${hostname}'
+    base_curl = 'curl --silent --disable --fail --max-time 5 --header "Authorization: Token ${secret}" "%s/sshkeys/?user=${user}&hostname=${hostname}'
```

Comment:
can you check the man page for curl? I think `--disable` has to be the first argument to `curl`.

Reply:
Yea it looks like it needs to be the first parameter, let me update it.

https://curl.se/docs/manpage.html#-q